### DISCUSSION TITLE
Unable to edit customer email (or order email)

### DISCUSSION THREAD

**Discussion by engram-design** [craftcms/commerce#2433] [Ideas]
For an order, it's currently not possible to edit a customer's email. The simple case of a mistyped email requires you to remove the customer from the order, and create a new one, which isn't ideal.

Thoughts on either being able to edit the customer email, or the email tracked against the order itself?

**Additional info**
- Craft version: 3.5.15.1
- Commerce version: 3.2.11

**Comment by pdaleramirez** [comment]
@engram-design This feature is in our list of features to work on. We can't give a specific timeline but we are hoping to include it Commerce 4.
